docs/fields/relationship.mdx

```diff
@@ -23,6 +23,7 @@ keywords: relationship, fields, config, configuration, documentation, Content Ma
 | **`name`** *         | To be used as the property name when stored and retrieved from the database.  |
 | **`*relationTo`** *  | Provide one or many collection `slug`s to be able to assign relationships to. |
 | **`hasMany`**        | Boolean when, if set to `true`, allows this field to have many relations instead of only one. |
+| **`maxDepth`**       | Sets a number limit on iterations of related documents to populate when queried. |
```

Comment:
We should link to the [Depth](https://payloadcms.com/docs/getting-started/concepts#depth) docs here.